package.json

```diff
@@ -1,70 +1,70 @@
-  },
-  "dependencies": {
-    "@apollo/client": "^3.3.14",
-    "@react-native-community/datetimepicker": "^3.4.7",
-    "@react-native-community/masked-view": "0.1.10",
-    "@react-navigation/bottom-tabs": "^5.11.10",
-    "@react-navigation/native": "^5.9.4",
-    "@react-navigation/stack": "^5.14.5",
-    "expo": "~40.0.0",
-    "expo-localization": "~9.1.0",
-    "expo-secure-store": "~9.3.0",
-    "expo-status-bar": "~1.0.4",
-    "formik": "^2.2.6",
-    "graphql": "^15.5.0",
-    "i18next": "^20.2.2",
-    "react": "16.13.1",
-    "react-dom": "16.13.1",
-    "react-i18next": "^11.8.15",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-40.0.1.tar.gz",
-    "react-native-dotenv": "^2.5.5",
-    "react-native-elements": "^3.4.1",
-    "react-native-gesture-handler": "~1.8.0",
-    "react-native-keyboard-aware-scroll-view": "^0.9.3",
-    "react-native-reanimated": "~1.13.0",
-    "react-native-screens": "~2.15.2",
-    "react-native-svg": "12.1.1",
-    "react-native-toast-message": "^1.4.9",
-    "react-native-web": "~0.14.13",
-    "string-to-color": "^2.2.2",
-    "yup": "^0.32.9"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.14.0",
-    "@graphql-codegen/cli": "^1.21.4",
-    "@graphql-codegen/introspection": "^1.18.1",
-    "@graphql-codegen/named-operations-object": "^1.18.1",
-    "@graphql-codegen/near-operation-file-preset": "^1.18.0",
-    "@graphql-codegen/typescript": "^1.21.1",
-    "@graphql-codegen/typescript-operations": "^1.17.16",
-    "@graphql-codegen/typescript-react-apollo": "^2.2.4",
-    "@types/react": "~16.9.35",
-    "@types/react-dom": "~17.0.3",
-    "@types/react-native": "~0.63.2",
-    "@typescript-eslint/eslint-plugin": "^4.22.0",
-    "@typescript-eslint/parser": "^4.22.0",
-    "eslint": "^7.26.0",
-    "eslint-config-airbnb": "^18.2.1",
-    "eslint-config-prettier": "^8.2.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-prettier": "^3.4.0",
-    "eslint-plugin-react": "^7.23.2",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "prettier": "^2.2.1",
-    "typescript": "~4.0.0"
-  },
-  "private": true
+    "main": "node_modules/expo/AppEntry.js",
+    "scripts": {
+        "start": "expo start",
+        "start:clear": "expo r -c",
+        "android": "expo start --android",
+        "ios": "expo start --ios",
+        "web": "expo start --web",
+        "eject": "expo eject",
+        "generate:types": "graphql-codegen -r dotenv/config",
+        "lint": "eslint --ext ts,tsx ./src",
+        "format": "prettier --check ./src"
+    },
+    "dependencies": {
+        "@apollo/client": "^3.3.17",
+        "@react-native-community/datetimepicker": "^3.4.7",
+        "@react-native-community/masked-view": "^0.1.11",
+        "@react-navigation/bottom-tabs": "^5.11.10",
+        "@react-navigation/native": "^5.9.4",
+        "@react-navigation/stack": "^5.14.5",
+        "expo": "^41.0.1",
```

Comment:
I guess updating the Expo SDK could have a strong impact - but if you tested it, it's fine 😃

Reply:
@vacatius/mobile-app-team I've tested it locally but it would be great if all of you could check out this branch an test it on your devices. Would like to see if this works on Android and other devices as well.